third_party/java/arcs/BUILD

```diff
@@ -13,3 +13,15 @@ alias(
     name = "arcs-wasm",
     actual = "//java/arcs/sdk:sdk-wasm",
 )
+
+# Arcs Datatypes for the JVM.
+alias(
+    name = "arcs-core-data",
```

Comment:
Instead of doing this, update //java/arcs/sdk:sdk (and sdk-wasm, etc.) to `export` the necessary deps.

Reply:
Thanks, that's a cleaner suggestion.